frontend/src/state/actions/advisorActions.ts

```diff
@@ -0,0 +1,32 @@
+import { createAction } from "typesafe-actions";
+import { Student } from "../reducers/advisorReducer";
+
+export const setEmail = createAction("advisor/SET_EMAIL", (email: string) => ({
```

Comment:
We might run into some issues if we use this action in a file that also uses the equivalent "user" action. Maybe `setAdvisorEmail` is better, but not sure if we'll run into this issue.